src/main/kotlin/no/nav/familie/ks/sak/kjerne/logg/LoggService.kt

```diff
@@ -0,0 +1,298 @@
+package no.nav.familie.ks.sak.kjerne.logg
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+import no.nav.familie.ks.sak.common.util.tilKortString
+import no.nav.familie.ks.sak.config.BehandlerRolle
+import no.nav.familie.ks.sak.config.RolleConfig
+import no.nav.familie.ks.sak.integrasjon.familieintegrasjon.domene.Arbeidsfordelingsenhet
+import no.nav.familie.ks.sak.kjerne.arbeidsfordeling.domene.ArbeidsfordelingPåBehandling
+import no.nav.familie.ks.sak.kjerne.behandling.Behandling
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingKategori
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat
+import no.nav.familie.ks.sak.sikkerhet.SikkerhetContext
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Service
+class LoggService(
```

Comment:
kanskje det er nyttig å kopiere metoden som du faktisk trenger nå og kan slippe andre. De andre metodene kan legges til ved behov. Det gjør to ting, en på github får vi en kommit med alle tilsvarende endringer og to slipper vi metodene som ikke gyldig for kontatntstøtte.

Reply:
Jeg har slettet en del metoder allerede som ikke var relevant for kontantstøtte.
De som ligger her trenger jeg til oppgavebenk og senere behandling

Reply:
Slettet noen flere nå, så nå bør bare grunnleggende ting ligge der